.github/workflows/github-ci.yml

```diff
@@ -0,0 +1,163 @@
+
+    - name: Test Core
+      run: |
+        cmake --build build -j 2 --target Test.netmeld-core
+        cd build
+        ctest -L ^Test.netmeld-core$
+    
+    - name: Package Core
+      run: |
+        cd ./scripts
+        bash ./make_netmeld-core_deb.sh
+        cd ..
+    
+    - name: Upload Deb Artifact
+      uses: actions/upload-artifact@v2
+      with:
+        name: netmeld-core-deb
+        path: scripts/*.deb
+  
+  
+  CoreModules:
+    needs: [Core]
+    
+    runs-on: ubuntu-latest
+    container: netmeld/netmeld-dev:latest
+
+    strategy:
+      matrix:
+        core-modules: [playbook-core, datalake-core]
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Configure All
+      run: cmake -S . -B build
+
+    - name: Build Core Modules
+      run: cmake --build build -j 2 --target netmeld-${{ matrix.core-modules }}
+
+    - name: Test Core Modules
+      run: |
+        cmake --build build -j 2 --target Test.netmeld-${{ matrix.core-modules }}
+        cd build
+        ctest -L ^Test.netmeld-${{ matrix.core-modules }}$
+    
+    - name: Package Core Modules
+      if: matrix.core-modules != 'datalake-core'
+      run: |
+        cd ./scripts
+        bash ./make_netmeld-${{ matrix.core-modules }}_deb.sh
+        cd ..
+    
+    - name: Upload Deb Artifact
+      if: matrix.core-modules != 'datalake-core'
+      uses: actions/upload-artifact@v2
+      with:
+        name: netmeld-${{ matrix.core-modules }}-deb
+        path: scripts/*.deb
+  
+  
+  Modules:
+    needs: [Core, CoreModules]
+    
+    runs-on: ubuntu-latest
+    container: netmeld/netmeld-dev:latest
+
+    strategy:
+      matrix:
+        modules: [core-bin, playbook, datalake, clw, exporters, fetchers, graphers, importers, inserters]
+       
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Configure All
+      run: cmake -S . -B build
+
+    - name: Build Modules
+      run: cmake --build build -j 2 --target netmeld-${{ matrix.modules }}
+
+    - name: Test Modules
+      run: |
+        cmake --build build -j 2 --target Test.netmeld-${{ matrix.modules }}
+        cd build
+        ctest -L ^Test.netmeld-${{ matrix.modules }}$
+    
+    - name: Package Modules
+      if: matrix.modules != 'datalake'
+      run: |
+        cd ./scripts
+        bash ./make_netmeld-${{ matrix.modules }}_deb.sh
+        cd ..
+    
+    - name: Upload Deb Artifact
+      if: matrix.modules != 'datalake'
+      uses: actions/upload-artifact@v2
+      with:
+        name: netmeld-${{ matrix.modules }}-deb
+        path: scripts/*.deb
+
+
+  PackageAll:
+    needs: [Core, CoreModules, Modules]
+
+    runs-on: ubuntu-latest
+    
+    steps:
+      - name: Download All Artifacts
+        uses: actions/download-artifact@v2
+      
+      - name: Prepare For Packaging
+        run: |
+          echo ::set-env name=DATETIME::$(date +%Y%m%d.%H%M%S)
+          find . -iname "*.deb" -print | zip --junk-paths artifacts -@
```

Comment:
Does GitHub require the packaging to be a `zip`?

Reply:
No, that was just what we were using before on GitLab. It can technically be any of the standard [Media Types](https://www.iana.org/assignments/media-types/media-types.xhtml) as far as I know. Should we change it to something else?

Reply:
Yeah, according to the [Release Asset](https://developer.github.com/v3/repos/releases/#upload-a-release-asset) docs it looks like it can be any of those media types.

Reply:
Hmmm, then we could publish the debs themselves.  Maybe we will think on that a bit.